readthedocs/organizations/forms.py

```diff
@@ -57,7 +62,11 @@ def clean_name(self):
         name = self.cleaned_data['name']
         if self.instance and self.instance.name and name == self.instance.name:
             return name
-        if Organization.objects.filter(slug=slugify(name)).exists():
+
+        potential_slug = slugify(name)
+        if not potential_slug:
+            raise forms.ValidationError(_('Invalid organization name'))
```

Comment:
```suggestion
            raise forms.ValidationError(_('Invalid organization name: no slug generated'))
```

This at least gives the user an idea how to fix it, but using `slug` is not great :/